Corona-Warn-App/src/deviceForTesters/java/de/rki/coronawarnapp/test/ccl/CCLTestViewModel.kt

```diff
@@ -27,20 +22,16 @@ class CCLTestViewModel @AssistedInject constructor(
     var selectedPersonIdentifier: PersonIdentifierSelection = PersonIdentifierSelection.All
     val dccWalletInfoList = dccWalletInfoRepository.personWallets.asLiveData2()
 
-    val personIdentifiers = combine(
-        vaccinationRepository.vaccinationInfos,
-        testCertificateRepository.cwaCertificates,
-        recoveryCertificateRepository.cwaCertificates
-    ) { vaccinatedPersons, tests, recoveries ->
-        val vaccinations = vaccinatedPersons.flatMap { it.vaccinationCertificates }.toSet()
-        val allCerts: Set<CwaCovidCertificate> = (vaccinations + tests + recoveries)
-        allCerts.map { it.personIdentifier }
-            .distinct()
-            .map { PersonIdentifierSelection.Selected(it) }
-            .toMutableList<PersonIdentifierSelection>()
-            .apply { add(0, PersonIdentifierSelection.All) }
-            .toList()
-    }.asLiveData2()
+    val personIdentifiers = certificateProvider.certificateContainer
+        .map { certificateContainer ->
+            certificateContainer.allCwaCertificates.map { it.personIdentifier }
+                .distinct()
+                .map { PersonIdentifierSelection.Selected(it) }
+                .toMutableList<PersonIdentifierSelection>()
+                .apply { add(0, PersonIdentifierSelection.All) }
+                .toList()
+        }
+        .asLiveData2()
```

Comment:
this has been removed in `2.19` please keep change minimal that address the bug ?